ansible/roles/mirrors/tasks/debian.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Gather LSB facts (for codename)
+  ansible.builtin.setup:
+    filter: ansible_lsb*
+  when: ansible_lsb is not defined
```

Comment:
I'm curious, when is `ansible_lsb` not defined? I'd expect it to be always gathered, or never gathered . . .